presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -198,6 +198,7 @@
 import static com.facebook.presto.operator.aggregation.MinByNAggregationFunction.MIN_BY_N_AGGREGATION;
```

Comment:
You could separate changes into separate commits have for:
 - `Apply` funciton
 - code generation 
 - semantic analyzer